### ISSUE TITLE
React 17 support?

### ISSUE THREAD

**Issue by simplenotezy** [codecks-io/react-sticky-box#70]
When installing in a react 17 library:

```
❯ npm install react-sticky-box
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: washworld@1.0.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^0.14.0 || ^15.0.0 || ^16.0.0" from react-sticky-box@0.9.3
npm ERR! node_modules/react-sticky-box
npm ERR!   react-sticky-box@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/mf/.npm/eresolve-report.txt for a full report.
```

npm ERR! A complete log of this run can be found in:

**Comment by DmitrySheklein** [comment]
➕

**Comment by DmitrySheklein** [comment]
@danielberndt, can you help ?

**Comment by kwcmwx** [comment]
with the parameter --force it worked

`npm install react-sticky-box --force`

**Comment by AntonioChagoya** [comment]
To force the installation may cause deployment issues, also looking for v17 support

**Comment by xreider** [comment]
It seems that it doesn't work on 17 React. It was the best library

**Comment by danielberndt** [comment]
v1.0.0 was just released with "react": ">=16.8.0" as peer dependency
